homeassistant/components/zwave/__init__.py

```diff
@@ -615,6 +623,19 @@ def change_association(service):
                          "target node:%s, instance=%s", node_id, group,
                          target_node_id, instance)
 
+    @asyncio.coroutine
+    def async_refresh_entity(service):
+        """Refresh values that specific entity depends on."""
+        entity_id = service.data.get(ATTR_ENTITY_ID)
+        async_dispatcher_send(
```

Comment:
This is refreshing only specific values and not the whole node, which is "heavier" procedure.